src/ontology/cl-edit.owl

```diff
@@ -31919,6 +31920,17 @@ AnnotationAssertion(rdfs:label obo:CL_4040005 "mesenchymal stem cell of apical p
 SubClassOf(obo:CL_4040005 obo:CL_0000134)
 SubClassOf(obo:CL_4040005 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003677))
 
+# Class: obo:CL_4042001 (TAC3-positive medium spiny neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727523") obo:IAO_0000115 obo:CL_4042001 "A nucleus accumbens shell and olfactory tubercle D1 medium spiny neuron that expresses TAC3. This medium spiny neuron has an enriched expression of the genes MPPED1, HPCAL1, MEIS3. ")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4042001 "https://orcid.org/0000-0002-0098-8958")
```

Comment:
Please add the ORCID as IRI.

Reply:
Do you mean this : https://orcid.org/0000-0002-0098-8958 ? 
or should I change it into a readable name following this link https://oboacademy.github.io/obook/howto/odk-add-orcidio-module/ ?

Reply:
The last commit is correct.